tests/integration/cattletest/core/test_container.py

```diff
@@ -264,7 +273,7 @@ def _assert_removed(container):
     return container
 
 
-def test_container_remove(admin_client, sim_context):
+def test_container_remove(admin_client, internal_test_client, sim_context):
```

Comment:
You utlimately return a version of the container scoped to the internal_test_client. Again, it might be better to return and admin_client scoped version of the container.